charts/redpanda/values.yaml

```diff
@@ -252,6 +271,120 @@ storage:
+  #
+  # Topic configuration flags override any cluster-level configuration. For example, on new topics,
+  # if `cloud_storage_enable_remote_write` is set to `true`, you can set `redpanda.remote.write` to `false`
+  # to turn it off for a particular topic. Or, if you have Tiered Storage turned off for the cluster, you can enable it
+  # for individual topics with the topic flags. When cluster-level properties are changed,
+  # the changes apply only to new topics, not existing topics.
+  #
+  # REF: https://docs.redpanda.com/docs/manage/tiered-storage/#enable-tiered-storage-for-a-topic
+  tieredConfig:
+    # Global flag that enables Tiered Storage feature if license is provided.
+    # Default is false.
+    cloud_storage_enabled: false
+    # Cluster wide config option to enable data upload from Redpanda to cloud storage if set to true.
+    cloud_storage_enable_remote_write: true
+    # Cluster wide config option to enable Redpanda to fetch data from cloud storage if set to true.
+    cloud_storage_enable_remote_read: true
+    # This configuration option is available starting from 22.3.X
+    #
+    # Possible values:
+    # `config` - If IAM roles are not available, specify credentials in the cluster configuration file.
+    # `aws_instance_metadata` - For an AWS EC2 instance, use the instance metadata API from AWS.
+    # `sts` - For AWS on Kubernetes, use the Secure Token Service (STS).
+    # `gcp_instance_metadata` - For a VM running on GCP, or for Google Kubernetes Engine (GKE), use the instance metadata API from GCP.
+    #
+    # REF: https://docs.redpanda.com/docs/manage/security/iam-roles/#configuring-iam-roles
+    cloud_storage_credentials_source: config
+    # Cloud storage region. Required.
+    cloud_storage_region: ""
+    # Cloud storage bucket name. Required.
+    cloud_storage_bucket: ""
+    # Cloud storage access key used to authenticated if `cloud_storage_credentials_source` is set to `config`
+    # cloud_storage_access_key: ""
+    # Cloud storage secret key used to authenticated if `cloud_storage_credentials_source` is set to `config`
+    # cloud_storage_secret_key: ""
+    # API endpoint.
+    #
+    # - For AWS, this can be left blank. It’s generated automatically using the region and bucket.
+    #
+    # - For Google Cloud Service, use storage.googleapis.com.
+    cloud_storage_api_endpoint: ""
+
+    # Maximum size of the disk cache used by Tiered Storage.
+    # Default is 20 GiB.
+    cloud_storage_cache_size: 20GiB
+    # The directory for the Tiered Storage cache. You must specify the full path. Default is: <redpanda-data-directory>/cloud_storage_cache.
+    # cloud_storage_cache_directory: ""
+    # The time, in milliseconds, between cache checks. The size of the cache can grow quickly,
+    # so it’s important to have a small interval between checks, but if the checks are too frequent,
+    # they consume a lot of resources.
+    # Default is 30000 ms.
+    cloud_storage_cache_check_interval: 30000
```

Comment:
For any setting that you're not setting to something non-default, please do not set it unless there is a strong reason to do so.  Letting redpanda handle the defaulting means we can evolve it in the future, whereas explicitly setting values will pin them forever to whatever the default in the helm chart was when the system was installed.

Reply:
Given my previous comment https://github.com/redpanda-data/helm-charts/pull/278#discussion_r1073454630 if I don't list the options users could not discover them easily if they start from artifact hub. 

To add more I mentioned release for Redpanda should have enough docs or release notes for k8s team to pick it up and adjust any property in helm.

Core team can evolve it in future I don't see any problem here. I would rather be always prescriptive.

Reply:
agreed, let's comment/document the defaults and make it explicit that we can override defaults.

Reply:
ok